### DISCUSSION TITLE
Excluding User Groups from Sales/Discounts

### DISCUSSION THREAD

**Discussion by andris-sevcenko** [craftcms/commerce#2299] [Ideas]
> Created by: Ahmad Dowla ([adowla@bespokeweb.nyc](adowla@bespokeweb.nyc)) on 2016/03/14 16:05:33 +0000
Votes at time of UserVoice import: 3

------
You can currently limit sales/discounts to specific user groups. What I'd like is the ability to exclude user groups.

Here's a use case: I have a client who does wholesale pricing at 50% off most products. In Craft Commerce, I would do this through a perpetual sale limited to a 'Wholesaler' user group. But if I want to simultaneously run a sale for regular customers--say a sitewide 15% discount--folks in the 'Wholesaler' user group would get a discount that's not exactly 50% (either 15% off 50% or 50% off 85%) because the sales would overlap.

Ideally, I would be able to create a sale that applies only to guests and users under a 'Customer' user group but not to users under the 'Wholesale' group.

**Comment by WHITE-developer** [comment]
@brandonkelly I see excluding user groups at discounts has been made since this discussion: https://github.com/craftcms/commerce/issues/220
But I see this option is not added to sales. Can this also be added to Sales?

Hope to hear from you!
